exporting/prometheus/prometheus.c

```diff
@@ -562,7 +655,8 @@ static void rrd_stats_api_v1_charts_allmetrics_prometheus(
                     } else {
                         // we need average or sum of the data
 
-                        time_t first_t = after, last_t = before;
+                        time_t first_t = instance->after;
```

Comment:
I always used in my life variable type as `type_t` thanks old C pattern that I believe is used yet, but here you are naming a variable like this, I will give a suggestion for we change the variable name removing `_t` to avoid confusion between data type and variable name. As you can see here on github, buf variable and type(`time_t`) are with the same color.

Reply:
We have 142 occurrences of such variables in our code. I renamed them in prometheus.c. I'll also do it soon in the rest of the exporting engine. Most of the variables will be deleted with the backends. We also have quite a lot of them in the registry.